chapter-16/game.js

```diff
@@ -0,0 +1,360 @@
+var simpleLevelPlan = `
```

Comment:
Please move all code that comes from the Eloquent JavaScript book into a `lib` subdirectory of `chapter-16`.

The `lib` directory name is commonly used for code that comes from external parties.